mdoc/Mono.Documentation/Updater/MsxdocDocumentationImporter.cs

```diff
@@ -125,14 +125,18 @@ public override void ImportDocumentation (DocsNodeInfo info)
                         }
                     default:
                         {
-                            bool add = true;
-                            if (child.NodeType == XmlNodeType.Element &&
-                                e.SelectNodes (child.Name).Cast<XmlElement> ().Any (n => n.OuterXml == child.OuterXml))
-                                add = false;
-                            if (add)
+                            var targetNodes = e.ChildNodes.Cast<XmlNode> ()
```

Comment:
@conceptdev or @lobrien ... was wondering if I could get a quick codereview on this logic change. I think what I've written is equivalent, and should have better performance, since it limits the calls to `.OuterXml` (which is [not a free call](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/Dom/XmlNode.cs#L952)) to only nodes who's name matches, and also "caches" the `.OuterXml` call of the node being considered for addition.

It passed the local test case for #159, and also passes the CI test suite ... just want a second opinion :)